boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -107,6 +167,13 @@ document.addEventListener("DOMContentLoaded", () => {
       onSearchLoadMoreButtonBehaviour(ssrDomEventString);
       const newEvents = parser.parseFromString(ssrDomEventString, "text/html");
       eventsContainer.append(...newEvents.body.children);
+
+      ////
+      const newlyAddedEvents = eventsContainer.querySelectorAll('.event');
+      setModalBehaviour(newlyAddedEvents)
```

Comment:
with this you re-add all event listeners to all events. so after a load-more, the original events have all listeners applied to them twice

Reply:
I couldn't find a better way, would it be possible to check it and give me feedback?